src/components/admin/MainIssueController.jsx

```diff
@@ -156,9 +157,8 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
```

Comment:
This is a clear name, thanks! I have a slight preference for action verbs and maybe calling it `getFirstRelativeURL`